### ISSUE TITLE
How do I change Relay email address?

### ISSUE THREAD

**Issue by CrushinIt17** [mozilla/fx-private-relay#855]
Hi,

Firefox Rely is a great feature, which I use all the time, keep up the amazing work Firefox Team!

With that said, I've changed my email across my Firefox account, however there seems to be no way to update the email address that Relay forwards to (my old email address, which I no longer have). Is there a way to update?

Warm regards,

A very happy Firefox supporter

**Comment by groovecoder** [comment]
Howdy. Relay forwards to the primary email address on your Firefox Account. You should be able to change that at https://accounts.firefox.com/settings.

**Comment by CrushinIt17** [comment]
@groovecoder hi luke, thank you for the reply. If fact, I did update my primary email address over a week ago and when I visit the Relay home page it still "Welcome's" me by my old email address and still forwards emails there as well (both old and newly generated Relays). Please help?

**Comment by groovecoder** [comment]
Hmm ... is it still "stuck" on your old email address? We had a bug in that code on our development server, but that should be fixed now?

**Comment by CrushinIt17** [comment]
@groovecoder thank you so much for your dedication on solving this bug. I just removed the Relay add-on, restarted Firefox, and reinstalled Relay - it is still 'stuck' on the old email address. Is try to help troubleshoot?

**Comment by CrushinIt17** [comment]
@groovecoder any updates on this bug by chance or are there any other things I can try to help troubleshoot? Thank you.

**Comment by groovecoder** [comment]
Sorry no updates yet. Let me see if our QA folks can help reproduce this so we can debug it.

**Comment by AOiegas** [comment]
Hey there. Unfortunately I cannot reproduce it, it works properly for me. I can add a secondary email, make it primary, and all the things that have to change change properly. Forwarded emails are sent to the new one, and I'm welcomed with the new email.

**Comment by CrushinIt17** [comment]
Thank you @AOiegas that did the trick! Many thanks to @groovecoder as well. 

Originally my Firefox account and Relay account were two separate email addresses, with the Relay address I no longer had access to and no way I could find to update it in Relay. I came back to follow up again, tried your suggestion above - under 'Manage your Firefox Account' I added a secondary email, set it as my primary and boom, Relay address was update with the new primary email address!

Thank you both for the great support!
